packages/cli/test/functional/test_site/expected/bugs/index.html

```diff
@@ -13,7 +13,7 @@
   <link rel="stylesheet" href="../markbind/fontawesome/css/all.min.css">
   <link rel="stylesheet" href="../markbind/glyphicons/css/bootstrap-glyphicons.min.css">
   <link rel="stylesheet" href="../markbind/css/octicons.css">
-  <link rel="stylesheet" href="../markbind/css/github.min.css">
+  <link rel="stylesheet" href="../markbind/css/vs2015.min.css">
```

Comment:
hmm, does this mean no light mode? =\

To clarify, what I meant is to use the light styles (`github.min.css`) during printing when dark mode is on (default) and under normal operation if dark mode is off (provide some way to toggle off dark mode)

e.g.

in site.json, maybe add a `style` key
```
{
  style: {
    codeTheme: "light" // implicitly dark by default
  }
}
```

> the problem shifts to us needing to provide the whole 'light' styling inside the rule.


regarding your earlier comment on duplication as well, you could try the `not print` operator https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries for the dark mode styles

Reply:
Oh I thought you want to set it as default (like the only one) as well from what I read on the issue 😅  I'll add the toggle on `site.json`